### ISSUE TITLE
Do we need to calculate l2_loss of the second GCN layer?

### ISSUE THREAD

**Issue by Orion-wyc** [dragen1860/GCN-PyTorch#4]
https://github.com/dragen1860/GCN-PyTorch/blob/623aa67f0d1c3384fe56f3a20e14e2c13845bcd9/model.py#L41-L54

It seems the L2 loss is only execute once on the first layer.

**Comment by c120129754** [comment]
definitely yes, this is a bug, ues this`    def l2_loss(self):
        layer = self.layers.children()
        layer = next(iter(layer))
        loss = None
        for layer in self.layers:
            for p in layer.parameters():
                if loss is None:
                    loss = p.pow(2).sum()
                else:
                    loss += p.pow(2).sum()
        return loss`
